Expensify/App
# [HOLD] [WIP] Introduce "Mark as unread" functionality in Expensify.cash

cc @roryabraham 

HELD on https://github.com/Expensify/Expensify.cash/pull/1977


### Details
The "New" marker has been modified so that it no longer disappears after a few seconds. It will behave like Slack's marker, which remains until the user leaves the chat. 

### Fixed Issues


### Tests
1. Log into e.cash and open a conversation with existing messages
2. On the hover menu of one of the messages, select "mark as unread":
<img width="1233" alt="Screen Shot 2021-03-16 at 3 43 38 PM" src="https://user-images.githubusercontent.com/19366280/111328469-8e7ccc00-866e-11eb-8f24-eed1d7d87d5b.png">
3. The `new` marker should show up over that message and the chat in the LHN should become bolded:
<img width="1602" alt="Screen Shot 2021-03-16 at 3 43 54 PM" src="https://user-images.githubusercontent.com/19366280/111328582-a6ece680-866e-11eb-90c7-b94ecb1e76b5.png">

5. Switch chats. It should remain bolded
6. Switch back to the original chat. The `new` marker should be correctly in place and the chat in the LHN should unbold in about 3 seconds. The `new` marker should remain in place after those 3 seconds.
7. Switch out and back in. the marker should be gone.
8. Do some regression testing to make sure the new marker shows correctly for actual new messages


### Tested On
